src/stores/api/manifest/stores.go

```diff
@@ -0,0 +1,92 @@
+package manifest
+
+import (
+	"context"
+
+	"github.com/consensys/quorum-key-manager/pkg/errors"
+	"github.com/consensys/quorum-key-manager/pkg/json"
+	authtypes "github.com/consensys/quorum-key-manager/src/auth/entities"
+	entities2 "github.com/consensys/quorum-key-manager/src/entities"
```

Comment:
entities2 ?

Reply:
entities are now split in the different domains but we have a tech debt to move them to a single folder: https://app.zenhub.com/workspaces/tanuki-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/431